pkgs/development/python-modules/pynest2d/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, python3, cmake, doxygen, gettext
+, pythonOlder, libnest2d, sip, clipper }:
+
+buildPythonPackage rec {
+  version = "4.8.0";
+  pname = "pynest2d";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "Ultimaker";
+    repo = "pynest2d";
+    rev = "4.8";
+    sha256 = "18dn92vgr4gvf9scfh93yg9bwrhdjvq62di08rpi7pqjrrvq2nvp";
+  };
+
+  disabled = pythonOlder "3.5.0";
+
+  buildInputs = [ python3 ];
+  propagatedBuildInputs = [ libnest2d sip clipper ];
+  nativeBuildInputs = [ cmake ];
+
+  CLIPPER_PATH = "${clipper.out}";
+
```

Comment:
The installation prefix problem has previously solved in libarcus: https://github.com/NixOS/nixpkgs/blob/37379c0215c2f9296db7189cef2bdc74c6af2138/pkgs/development/python-modules/libarcus/default.nix#L22-L24